tests/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_subdirectory( cppVersion )
-add_subdirectory( city )
+add_subdirectory( City )
```

Comment:
@kermit10000000 I would suggest `city` begin with *lowercase*.
Other directories in test dir begin with lowercase, so in my opinion we should not break the convention.
...but it is up to you, please, merge it and this can be fix later or fix it with this pull request.

Reply:
add_subdirectory( FlagParser ) does not start from lowercase either so I will merge it now and maybe we will make a common decision towards that together